pip/commands/install.py

```diff
@@ -266,6 +276,7 @@ def run(self, options, args):
                     "continue."
                 )
             install_options.append('--home=' + temp_target_dir)
+            install_options.append('--prefix=')
```

Comment:
To my understanding the exception only avoids both options being set at the pip command line.
This line ensures that a `prefix`     .